src/styles/general.scss

```diff
@@ -0,0 +1,3 @@
+$mobile-width: 480px;
+$tablet-width: 768px;
+$desktop-width: 1024px;
```

Comment:
Is there any meaningful difference between `common.scss` and `general.scss`? As a user I would not understand it. Maybe let's put these in `common.scss`?